w_flux_codemod/lib/src/action_v2_suggestor.dart

```diff
@@ -51,48 +45,60 @@ mixin ActionV2Migrator on AstVisitingSuggestor {
 class ActionV2DispatchMigrator extends RecursiveAstVisitor
     with AstVisitingSuggestor, ActionV2Migrator {
   @override
-  bool shouldMigrate(InvocationExpression node) =>
-      node.staticInvokeType != null &&
-      node.argumentList.arguments.isNotEmpty &&
-      node.function.staticParameterElement.name == 'dispatch';
+  bool shouldMigrate(node) {
+    if (node is FunctionExpressionInvocation) {
+      final name = node.function.staticType?.element?.displayName;
+      // the type migration should have happened prior to this suggestor.
+      return name == 'ActionV2';
```

Comment:
Should we move this suggestor to an action_v2_migrate_step_3? Or does placing it at the end of list enough to ensure that it will run last?

Reply:
Hmm.. well I guess my understanding of how `runInteractiveCodemodSequence` is wrong (or there's a bug), because I tried this and it didn't work. Had to separate them out to two separate calls to `runInteractiveCodemod` like so:

```dart
  exitCode = await runInteractiveCodemod(
    filePathsFromGlob(Glob('**.dart', recursive: true)),
    aggregate([
      ActionV2FieldAndVariableMigrator(),
      ActionV2ParameterMigrator(),
      ActionV2ReturnTypeMigrator(),
      ActionV2SuperTypeMigrator(),
    ]),
    args: args,
  );
  if (exitCode != 0) {
    return;
  }
  exitCode = await runInteractiveCodemod(
    filePathsFromGlob(Glob('**.dart', recursive: true)),
    ActionV2DispatchMigrator(),
    args: args,
  );
```